CMakeLists.txt

```diff
@@ -155,7 +155,11 @@ endif()
 list(APPEND qjs_defines _GNU_SOURCE)
 list(APPEND qjs_libs qjs ${CMAKE_DL_LIBS})
 if(NOT MSVC)
-    list(APPEND qjs_libs m pthread)
+    if (NOT ANDROID)
```

Comment:
While this works, I think using CMAKE_THREAD_LIBS_INIT feels more future proof: https://cmake.org/cmake/help/latest/module/FindThreads.html